card_reader_server.py

```diff
@@ -60,10 +61,11 @@ def update(self, observable, cards):
             connection.transmit([0xFF, 0x00, 0x40, p2, 0x04, 0x02, 0x00, 0x01, 0x01])
 
             # Pipe card ID to the enabled output(s)
+            gtid = "{}{}".format(prefix, str(file_data_byte[0])) if prefix is not None else str(file_data_byte[0])
```

Comment:
f strings in Python 3 are much prettier than this: `f"{variableOrExpression} is cool"`